users.go

```diff
@@ -115,6 +115,7 @@ type ListUsersOptions struct {
 	OrderBy              *string    `url:"order_by,omitempty" json:"order_by,omitempty"`
 	Sort                 *string    `url:"sort,omitempty" json:"sort,omitempty"`
 	External             *bool      `url:"external,omitempty" json:"external,omitempty"`
+	ExcludeInternal      *bool      `url:"exclude_internal,omitempty" json:"exclude_internal,omitempty"`
```

Comment:
This looks good, but it's an option that is also available for normal users. So this should be added below line 106 I guess?

Reply:
You are right, I can verify that the GitLab documentation also says it is also available for https://docs.gitlab.com/ee/api/users.html#for-normal-users not just for admins. Force pushed the branch